CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [3.0.1] - 2020-09-15
```

Comment:
The links at the bottom of the changelog still need to be updated with this release

Reply:
Good catch, thanks! 😅